sklearn/preprocessing/tests/test_data.py

```diff
@@ -1270,6 +1270,39 @@ def test_normalize():
     assert_raises(ValueError, normalize, [[0]], norm='l3')
 
 
+def test_normalize_l1():
+    rs = np.random.RandomState(0)
+    X_dense = rs.rand(10, 5)
+    X_sparse = sparse.csr_matrix(X_dense)
+    ones = np.ones((10))
+    for X in (X_dense, X_sparse):
+        for dtype in (np.float32, np.float64):
+            X = X.astype(dtype)
+            X_norm = normalize(X, norm='l1')
+            assert_equal(X_norm.dtype, dtype)
+
+            X_norm = toarray(X_norm)
+            row_sums = np.abs(X_norm).sum(axis=1)
+            assert_array_almost_equal(row_sums, ones)
+
+
+def test_normalize_l2():
```

Comment:
Hello @maniteja123 , thanks for your review.
there's also a difference [here]. (https://github.com/scikit-learn/scikit-learn/pull/6539/files#diff-9391886e2d1a9da2c6e629c950e990efR1301)

But yeah, I can use a `if` to do that.
I'll wait for reviewer's opinion.